packages/eslint-plugin/docs/rules/restrict-template-expressions.mdx

```diff
@@ -148,8 +148,6 @@ const msg1 = `arg = ${arg}`;
 
 {/* insert option description */}
 
-Whether to allow additional types in template expressions.
```

Comment:
Option's description is slightly different: `Types to allow in template expressions.`.